packages/dx-react-grid/docs/guides/getting-started.md

```diff
@@ -49,7 +57,9 @@ Use the Table plugin to display the data as a simple table:
 ```jsx
 import {
   Grid, Table, TableHeaderRow
-} from '@devexpress/dx-react-grid-bootstrap3'/* or '@devexpress/dx-react-grid-material-ui' */;
+} from '@devexpress/dx-react-grid-bootstrap4';
```

Comment:
I don't understand logic. We put the first MUI everywhere. Why do we put the first BS4 here?